lib/src/fwupd_device.dart

```diff
@@ -117,6 +117,15 @@ class FwupdDevice {
   /// One line device summary.
   final String? summary;
 
+  /// Update error.
+  final String? updateError;
```

Comment:
Missing tests for updateError and updateImage.

Reply:
Added to the device signals test